src/codegen/mod.rs

```diff
@@ -2745,7 +2745,7 @@ mod utils {
     }
 
     pub fn fnsig_return_ty(ctx: &BindgenContext,
-                           sig: &super::FunctionSig)
+                           sig: &FunctionSig)
```

Comment:
If we are going to support older rustc, we should add the oldest supported version to our travis ci matrix.